test/test_gpu2.cpp

```diff
@@ -9201,84 +9201,83 @@ TEST_F(NVFuserTest, FusionTestWarpSoftMax_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {ref_output}, __LINE__, __FILE__);
 }
 
-// NOTE: disabling 1133 test for cuda12.2 failure. See issue:
```

Comment:
We should't comment out a test. We should use `GTEST_SKIP`. I do need to change the check below to
```C++
        TORCH_CHECK(
            !alloc->buffer()->isA<TensorView>(), "Invalid allocation detected");
```
so that it does not fail earlier than `testValidate`.